_rules/SC4-1-1-unique-id.md

```diff
@@ -42,16 +42,22 @@ There are no major accessibility support issues known for this rule.
 
 ### Passed
 
+#### Pass example 1
+
```

Comment:
Lets add descriptions here too.

Reply:
I have created an issue for this, as I was not the author for the same. https://github.com/auto-wcag/auto-wcag/issues/228